ci.py

```diff
@@ -14,7 +14,7 @@
 from os import environ
 from pathlib import Path
 from subprocess import PIPE, run
-from sys import exit, stderr
+from sys import stderr
```

Comment:
`exit` is used on line 79 ... Which linter recommended this?

Reply:
`exit` is already available as a builtin, and does what `sys.exit` does.  pylint rightly points out that this was shadowing/redefining a builtin.  https://docs.python.org/3/library/constants.html#quit